3/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh

```diff
@@ -247,14 +257,26 @@ zookeeper_generate_conf() {
     zookeeper_conf_set "${ZOO_CONF_DIR}/log4j.properties" zookeeper.console.threshold "$ZOO_LOG_LEVEL"
 
     # Add zookeeper servers to configuration
+    idFlag=false
+    if [[ "$ZOO_SERVERS" == *"::"* ]]; then
+        idFlag=true
+    fi
     read -r -a zookeeper_servers_list <<< "${ZOO_SERVERS//[;, ]/ }"
     if [[ ${#zookeeper_servers_list[@]} -gt 1 ]]; then
-        local i=1
-        for server in "${zookeeper_servers_list[@]}"; do
-            info "Adding server: ${server}"
-            zookeeper_conf_set "$ZOO_CONF_FILE" server.$i "${server};${ZOO_PORT_NUMBER}"
-            (( i++ ))
-        done
+        if [[ "${idFlag}" == "false" ]]; then
```

Comment:
To be consistent with my previous suggestion:
```suggestion
        if is_boolean_yes "$server_id_with_jumps"; then
```